### PR TITLE
pipewire, pw-dump, pw-profiler: add pages

### LINKED ISSUE #11747: Let's document: PipeWire
### Command description

PipeWire is quite stable now and even reached 1.0 recently. For now we have just a few commands documented, it would be nice if we documented everything :smiley:

### Documentation

https://docs.pipewire.org/page_programs.html

### Platform

Linux

### Additional information

_No response_

### Commands

- [ ] pipewire
- [X] pw-cat
- [X] pw-cli
- [ ] pw-config
- [ ] pw-dot
- [ ] pw-dump
- [ ] pw-jack
- [X] pw-link
- [X] pw-loopback
- [ ] pw-metadata
- [ ] pw-mididump
- [ ] pw-mon → #11744
- [ ] pw-profiler
- [ ] pw-reserve
- [ ] pw-top
- [ ] pw-v4l2
- [ ] spa-acp-tool
- [ ] spa-inspect
- [ ] spa-json-dump
- [ ] spa-monitor
- [ ] spa-resample

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Addresses #11747. I'll translate them too when I have time and maybe even add some more pages.

### CODE REVIEW

File: pages/linux/pipewire.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# pipewire
+
+> Starts the PipeWite daemon.
```
Comment (kbdharun) [inline pages/linux/pipewire.md:3]:
```suggestion
> Start the PipeWite daemon.
```

File: pages/linux/pipewire.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`pipewire`
+
+- Use a different configuration file:
+
+`pipewire --config={{path/to/file.conf}}`
+
+- Set a verbosity level (error, warn, info, debug or trace):
+
+`pipewire -{{v|vv|...|vvvvv}`
```
Comment (kbdharun) [inline pages/linux/pipewire.md:16]:
```suggestion
`pipewire -{{v|vv|...|vvvvv}}`
```

File: pages/linux/pw-dump.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# pw-dump
+
+> Dump PipeWire current state as JSON, including the information on nodes, devices, modules, ports, and other objects.
```
Comment (kbdharun) [inline pages/linux/pw-dump.md:3]:
```suggestion
> Dump PipeWire's current state as JSON, including the information on nodes, devices, modules, ports, and other objects.
```

File: pages/linux/pw-profiler.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# pw-profiler
+
+> Profile a local or remote instance.
+> More information: <https://docs.pipewire.org/page_man_pw-profiler_1.html>.
+
+- Profile the default instance, logging to profile.log (gnuplot files and a .html file for result visualizing are also generated):
```
Comment (kbdharun) [inline pages/linux/pw-profiler.md:6]:
```suggestion
- Profile the default instance, logging to `profile.log` (`gnuplot` files and a `.html` file for result visualizing are also generated):
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page (please update the same in translations too).

File: pages/linux/pipewire.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# pipewire
+
+> Start the PipeWite daemon.
```
Comment (acuteenvy) [inline pages/linux/pipewire.md:3]: typo
```suggestion
> Start the PipeWire daemon.
```

File: pages/linux/pipewire.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Start the PipeWire daemon:
+
+`pipewire`
+
+- Use a different configuration file:
+
+`pipewire --config={{path/to/file.conf}}`
+
+- Set a verbosity level (error, warn, info, debug or trace):
```
Comment (acuteenvy) [inline pages/linux/pipewire.md:14]:
```suggestion
- Set the verbosity level (error, warn, info, debug or trace):
```

**Comment by vitorhcl** [comment]
If it is fine, I think we can merge this and translate pw-profiler to pt_BR in another PR.

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. The English pages LGTM, after @isaacvicente's review of the pt_BR pages this PR is GTG.

**Pull request merged**